Galaxus.Functional/(Option)/None.cs

```diff
@@ -3,30 +3,38 @@
 namespace Galaxus.Functional
 {
     /// <summary>
-    /// Denotes the absence of <b>Some</b>.
-    /// This type is most often used to implicitly convert from its value to an <see cref="Option{T}"/>.
+    ///     Denotes the absence of <b>Some</b>.
+    ///     This type is most often used to implicitly convert from its value to an <see cref="Option{T}" />.
     /// </summary>
     public struct None : IEquatable<None>
     {
         /// <summary>
-        /// The <b>None</b> value.
+        ///     The <b>None</b> value.
         /// </summary>
         public static readonly None Value = default;
 
         /// <inheritdoc />
         public override bool Equals(object other)
-            => other is None;
```

Comment:
tl;dr: Ease of automation, basically.

It's a hard pill to swallow for me, since I like the expression-bodied syntax, but it's hard to configure the cleanup to "do expression-bodies where it's short and succint but use statement bodies everywhere else". With what I know about editorconfigs, it's either "use expression bodies *wherever* possible" or "don't use expression bodies at all" for a consistent style.

I've opted to "don't use expression bodies at all" - to make all functions and properties look alike.